geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSMoveIntegrationTest.java

```diff
@@ -34,16 +33,19 @@
 import org.apache.geode.redis.ConcurrentLoopingThreads;
 import org.apache.geode.redis.RedisIntegrationTest;
 import org.apache.geode.redis.internal.RedisConstants;
-import org.apache.geode.test.awaitility.GeodeAwaitility;
 
 public abstract class AbstractSMoveIntegrationTest implements RedisIntegrationTest {
```

Comment:
It would be good to also have a test showing the behaviour when the source and destination key do not map to the same slot.

Reply:
Is this supposed to be when the source and destination key do map to the same slot? (Use the same key for destination and source?)

Reply:
No, different slots. Currently all the tests are using keys which have the same tag, {user1}, which means that they'll all map to the same slot (that's the purpose of using tags). It would be good to have a test where one key is `{tag1}key` and the other key is `{tag2}key` so that the slots they map to are different and we can see the behaviour for that case is correct.